### ISSUE TITLE
Random crash on 1st load of the day.

### ISSUE THREAD

**Issue by RaliiTheAvali** [Neos-Metaverse/NeosPublic#2442]
## Describe the bug
When ever i launch Neos for the 1st time in a day It will Crash/close out within the 1st min.

## Relevant issues
N/A

## To Reproduce
[Steps to reproduce the behavior:
1. Launch Neos via the Steam Lib. and selecting the " Launch Neos in Steam VR mode"
2. Let Neos start up.
3. wait for it to randomly close out with no crash indication or error.

## Expected behavior
 To not crash 1st time start up each day?

## Log Files
[RALII-PC - 2021.6.8.685 - 2021-06-08 19_05_50.log](https://github.com/Neos-Metaverse/NeosPublic/files/6620088/RALII-PC.-.2021.6.8.685.-.2021-06-08.19_05_50.log)
 NO UNITY CRASH LOG GETS CREATED

## Screenshots / Video
N/A

## Bug information (please complete the following information):
 - How often does it happen: [Always-Only Once a day]
 - Does the bug persist after restarting Neos? [No Not after the 1st 'crash' of the day]
 - Neos Version: [2021.6.8.685]
 - Neos Platform: [Windows]
 - Regression: [N/A only now started paying attention to the random crashes]

## Reporters:
Ralii Ralii#2181

**Comment by Frooxius** [comment]
I don't see too much specific in the log unfortunately. It seems to crash when a LIV virtual camera is created, could you try disabling that and see if that prevents the crash? Might be something odd going with SteamVR initialization process.

**Comment by Ardescar** [comment]
I experienced something that i till now havnt posted in git ... i only wrote it to shifty and was discussing it to reproduce ....

What i so far found :::
when i startup steam vr and then neos (it does seem also to work only mainly for the first time a day - till now not fully tested)

- i was waiting for neos to fully boot up - and/or was working in desktop (F8) abit - but desktop is not needed for crash.
- then after neos is running fine and i was ready with desktop work i was changing to VR finally 
- so the headset did turn on everything fine (Index)
- when i was booting up the controllers - neos does close - crash without crash report

It could be related to Ralii's issue. Because i have the feeling that this also happens more often if its the first bootup.
If i switch everything off and wait a bit i can reproduce it quiet good.

## Log Files
[ARDES - 2021.5.29.56 - 2021-05-29 09_44_37.log](https://github.com/Neos-Metaverse/NeosPublic/files/6564265/ARDES.-.2021.5.29.56.-.2021-05-29.09_44_37.log)

**Comment by RaliiTheAvali** [comment]
> 
> 
> I experienced something that i till now havnt posted in git ... i only wrote it to shifty and was discussing it to reproduce ....
> 
> What i so far found :::
> when i startup steam vr and then neos (it does seem also to work only mainly for the first time a day - till now not fully tested)
> 
>     * i was waiting for neos to fully boot up - and/or was working in desktop (F8) abit - but desktop is not needed for crash.
> 
>     * then after neos is running fine and i was ready with desktop work i was changing to VR finally
> 
>     * so the headset did turn on everything fine (Index)
> 
>     * when i was booting up the controllers - neos does close - crash without crash report
> 
> 
> It could be related to Ralii's issue. Because i have the feeling that this also happens more often if its the first bootup.
> If i switch everything off and wait a bit i can reproduce it quiet good. But it depends on if Ralii does do the same things like i do.
> ## Log Files
> 
> [ARDES - 2021.5.29.56 - 2021-05-29 09_44_37.log](https://github.com/Neos-Metaverse/NeosPublic/files/6564265/ARDES.-.2021.5.29.56.-.2021-05-29.09_44_37.log)

actually now that you mention it, it does seem to happen after i turn on the Index controllers as well as my trackers.

**Comment by Frooxius** [comment]
Can you check the SteamVR log and see if there's anywhere? You can get it if you right click the SteamVR window and go to Developer -> Web Console.

Can you switch to SteamVR beta if you're using normal or vice versa and see if that changes anything?

**Comment by RaliiTheAvali** [comment]
> 
> 
> Can you check the SteamVR log and see if there's anywhere? You can get it if you right click the SteamVR window and go to Developer -> Web Console.
> 
> Can you switch to SteamVR beta if you're using normal or vice versa and see if that changes anything?

when i get home i can give it a try

**Comment by Ardescar** [comment]
Here is my SteamVRLog - i was home a bit earlier than Ralii <3 *hugs*

[steamvrwebconsole.txt](https://github.com/Neos-Metaverse/NeosPublic/files/6624939/steamvrwebconsole.txt)

**Comment by Ardescar** [comment]
And it also crashes in Steam VR Beta

[steamvrbetawebconsole.txt](https://github.com/Neos-Metaverse/NeosPublic/files/6624995/steamvrbetawebconsole.txt)

**Comment by RaliiTheAvali** [comment]
Started up today and as soon as i turned on the Right controller it closed Neos out with out a crash log, but heres one of the Logs for the Steam Beta 1.18.1 
[SteamVR log Beta 1.18.1 log.txt](https://github.com/Neos-Metaverse/NeosPublic/files/6634993/SteamVR.log.Beta.1.18.1.log.txt)


I only have the Beta of steam vr ( don't know how to switch beta came by default wasn't even aware there was a non-beta.)


Also have LIV but Liv but its also off by default.
